content/200-orm/500-reference/200-prisma-cli-reference.mdx

```diff
@@ -402,7 +402,7 @@ generator client {
 | `--data-proxy`      | No       | The `generate` command will generate Prisma Client for use with [Prisma Accelerate](/accelerate) prior to Prisma 5.0.0. Mutually exclusive with `--accelerate` and `--no-engine`.                                                   |
 | `--accelerate`      | No       | The `generate` command will generate Prisma Client for use with [Prisma Accelerate](/accelerate). Mutually exclusive with `--data-proxy` and `--no-engine`. Available in Prisma 5.1.0 and later.                                    |
 | `--no-engine`       | No       | The `generate` command will generate Prisma Client without an accompanied engine for use with [Prisma Accelerate](/accelerate). Mutually exclusive with `--data-proxy` and `--accelerate`. Available in Prisma ORM 5.2.0 and later. |
-| `--no-hints`        | No       | The `generate` command will generate Prisma Client without usage hints being printed to the terminal. Available in Prisma ORM 5.16.0 and later.                                                                                     |
+| `--no-hints`        | No       | The `generate` command will generate Prisma Client without usage hints, surveys or ads being printed to the terminal. Available in Prisma ORM 5.16.0 and later.                                                                                     |
```

Comment:
Don't like the word "ads" much. Can we replace with the following?

The `generate` command will generate Prisma Client without usage hints, surveys, or info banners. Available in Prisma ORM 5.16.0 and later.